### ISSUE TITLE
Modal design guidelines: aria-label cannot be placed on i elements

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-org#3198]
Work towards #2920 

The icons on the Modal design guidelines page should not have an `aria-label` attribute on the `<i>` elements, and instead should be moved to the `<span>` wrapping them.

**Comment by RuiBrian** [comment]
I can be assigned to this issue!
